src/pages/about.tsx

```diff
@@ -1,128 +1,114 @@
 import { Link } from 'next-theme-ui';
-import {
-  Flex,
-  Grid,
-  Heading,
-  Paragraph,
-  Text,
-  Link as ThemeUILink,
-} from 'theme-ui';
+import { Box, Container, Flex, Grid, Heading, Paragraph, Text } from 'theme-ui';
 import { ContactPerson } from '../components/about/contact-person';
-import { Booking } from '../components/booking';
+import { BackgroundGraphic } from '../components/background-graphic';
 import { Layout } from '../components/layout/layout';
+import { PageSection } from '../components/page-section';
 
 const About: React.FC = () => {
   return (
     <Layout>
-      <Grid sx={{ gap: 5 }}>
-        <Grid sx={{ gap: 3 }}>
-          <Heading variant="title">Om oss</Heading>
-          <Paragraph variant="excerpt">
+      <PageSection>
+        <BackgroundGraphic
+          element="element2"
+          positionx="left"
+          positiony="1040px"
+        />
+
+        <Container variant="readable">
+          <Heading variant="titlenobold" sx={{ pb: '40px' }}>
```

Comment:
```suggestion
          <Heading variant="titlenobold" sx={{ pb: '40px' }}>
```

This and the one below 👇 should probably be replaced with the size array.